Automattic/wp-calypso
# wpcom-block-editor - add fill for closing site editor.

#### Changes proposed in this Pull Request

* Adds fill to fix "Back to dashboard" button in site editor navigation panel when used through calypso.

#### Testing instructions



* Sync changes from this PR to your sandbox using `yarn dev --sync` from the `apps/wpcom-block-editor` directory
* Sandbox widgets.wp.com
* Load the site editor from calypso and verify 'back to dashboard' leads back to calypso as expected.
* Smoke test the post editor.

Fixes #46822
